miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/JsonArrayCollector.java

```diff
@@ -0,0 +1,46 @@
+package uk.ac.bbsrc.tgac.miso.webapp.util;
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.function.BiConsumer;
+import java.util.function.BinaryOperator;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.stream.Collector;
+
+public class JsonArrayCollector implements Collector<java.lang.String, StringBuilder, java.lang.String> {
+
+  @Override
+  public BiConsumer<StringBuilder, String> accumulator() {
+    return (sb, metrics) -> {
+      if (sb.length() == 0) {
+        sb.append(metrics);
+      } else {
+        sb.append(".concat(");
```

Comment:
Could this convert to proper JSON instead? Storing JSON in the DB would be preferable to JS